### PR TITLE
Bugfixes general / best format

### PR DESCRIPTION
**Pull request by rokdd**
Fixes the problem, that when a format is given which is not in the list, that best format is not working. All unknown formats get last priority.

### CODE REVIEW

**Review by Josef-Friedrich — CHANGES_REQUESTED** [review]
The you for your pull request. Can you please remove two lines, so that the tests are passing again.

File: audiorename/audiofile.py [hunk 1]
```diff
@@ -319,6 +320,8 @@ def count(key):
         return
 
     if job.output.debug:
+        return
+        # following line throws errors... maybe meta fields is not correct set?
```
[the remark this answers is not in the document]
Comment (rokdd) [reply audiorename/audiofile.py]: Well I understand.. I changed it back.

**Comment by Josef-Friedrich** [comment]
Great! Thank!

By the way: a minor flake8 issue. I will fix it for you  

audiorename/audiofile.py:165:1: W293 blank line contains whitespace

**Comment by rokdd** [comment]
Great thanks.. can I do flake with a tox command? Because I guess for next 2 PR will it be also helpful

**Comment by Josef-Friedrich** [comment]
Yes `tox -e flake8`

**Pull request merged**